manifest.yml

```diff
@@ -5,9 +5,9 @@ description: '`ImageTorchEncoder` encodes data from a `ndarray` into a d-dimensi
 author: Jina AI Dev-Team (dev-team@jina.ai)
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.1
```

Comment:
I didn't see the point to have semantic versioning for Jina Hub. Let's use the simplest incremental versioning.
```
version: 2
```

Reply:
ok